### DISCUSSION TITLE
Training Materials

### DISCUSSION THREAD

**Discussion by keurek** [NREL/ReEDS-2.0#21] [Q&A: Execution] [answered]
Are there training materials for how to run ReEDS?

**Comment by keurek** [comment, marked as answer]
A ReEDS training video (recorded in July 2020 and based on the 2019 version of ReEDS) is available on the NREL YouTube channel at https://youtu.be/Cdo27F18AZA. In addition, the Open-Access ReEDS Webinar from October 2019 gives an overview of the 2019 ReEDS model and how it works (https://www.youtube.com/watch?v=QpRtvs_0kkA).

**Comment by cvcastro** [comment]
How to remove me from this list-serv?


*Cyndi Vail Castro, PE*
*www.cvcastro.com* <http://www.cvcastro.com>

**Comment by keurek** [comment]
Hi Cyndi,
You have a couple of options:
1. You may unsubscribe from the email notifications by adjusting your notification settings on GitHub.
https://docs.github.com/en/github/managing-subscriptions-and-notifications-on-github/setting-up-notifications/configuring-notifications

2. You may remove yourself as a collaborator from the ReEDS_OpenAccess repository.
https://docs.github.com/en/github/setting-up-and-managing-your-github-user-account/managing-access-to-your-personal-repositories/removing-yourself-from-a-collaborators-repository

I hope this helps.

-Kelly
